templates/footer.html

```diff
@@ -42,10 +42,13 @@ <h3 class="contact-title">Keep in touch</h3>
                   <a href="https://hub.docker.com/u/reanahub/">@reanahub</a>
                 </td>
               </tr>
-
               <tr>
                 <td class="contact-list-header"><i class="fas fa-book"></i> Docs</td>
-                <td><a href="http://docs.reana.io/">Read The Docs</a></td>
+                <td><a href="https://docs.reana.io/">docs.reana.io</a></td>
+              </tr>
+              <tr>
+                <td class="contact-list-header"><i class="fab fa-discourse"></i> Forum</td>
```

Comment:
Shall we also add Blog link to the footer?  It would go nicely between Docs and Forum... at the price of making the footer even more thicker than it is now.  But I guess we can live with that; having more complete link factory at the project footer seems more important.